Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,21 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const totalEarnings = tasks
+    .map(task => task.duration / 60)
+    .filter(rate => rate >= 2)
+    .map(rate => rate * hourlyRate)
+    .reduce((acc, curVal) => Math.round((acc + curVal) * 100) / 100);
+  return totalEarnings;
 }
```

Comment:
Hi @BassamHajar - I liked how you used method chaining here to accomplish the desired results. I would like to add one thing here though. Although it might seem a small thing but giving appropriate names to variables inside methods like `map`, `filter` and `reduce` is quite important. I see that you have done a mix of both.

For example, the `map` method uses `task` as an internal variable correctly. However, `filter` method uses `rate` as the variable name when `hour` might have been more appropriate because we are filtering out hours here. Similarly, inside `reduce`, you have used `acc` and `curVal` which might have been coming from the APIs of the reduce method but having `total`, `amount` would have been more appropriate. Don't you think? 🙂 

Please be mindful of variable naming conventions. Even though they are small things but they happen to make you lot better.